### DISCUSSION TITLE
How to make an accessible list box?

### DISCUSSION THREAD

**Discussion by clmnin** [radix-ui/primitives#3018] [Help]
I have a large list that needs to be keyboard accessible. Currently, I can tab between list items, but a better interaction/UX would be to use the arrow keys.

React Aria Components have a list box component - https://react-spectrum.adobe.com/react-aria/ListBox.html. Do we have something like this, or can we build something similar using Radix UI Primitives?

**Comment by alissawix** [comment]
the best option I've found is toggle group component

**Comment by ted-palmer** [comment]
I made a demo of using the toggle group component as a list box: [https://codesandbox.io/p/github/ted-palmer/listbox-demo/main?import=true](https://codesandbox.io/p/github/ted-palmer/listbox-demo/main?import=true)
